arbitrum-docs/partials/_contribute-docs-partial.md

```diff
@@ -0,0 +1,258 @@
+
+:::
+
+
+Usage:
+
+```
+:::caution UNDER CONSTRUCTION
+
+The following steps are under construction and will be updated with more detailed guidance soon. Stay tuned, and don't hesitate to click the `Request an update` at the top of this document if you have any feedback along the way.
+
+:::
+```
+
+<br/>
+
+#### Community member contribution banner
+
+Example:
+
+:::info Community member contribution
+
+The following document was contributed by @todo-twitter-handle. Give them a shoutout if you find it useful!
+
+:::
+
+
+Usage:
+
+```
+:::info Community member contribution
+
+The following document was contributed by @todo-twitter-handle. Give them a shoutout if you find it useful!
+
+:::
+```
+
+
+
+### Organization conventions
+
+Our published docs are generally organized like this in the sidebar:
+
+```
+📄 Gentle introduction
+📄 Quickstart
+📂 How-tos
+  📄 Write a smart contract
+📂 Concepts
+  📄 Smart contract
+📁 Reference
+📄 Troubleshooting
+📄 FAQ
+📄 Glossary
+📄 Contribute docs
+📂 Third-party content
+  📁 Product A
+  📂 Product B
+    📄 How to test your smart contract using Product B
+```
+
+ - This isn't a strict requirement; it's a loose guideline.
+ - Not all product areas have all types of content.
+ - Deviating from this pattern can make important content more discoverable to readers. For example, [Developer tools and resources](https://developer.arbitrum.io/for-devs/dev-tools-and-resources/overview) is a root-level collection of reference-type content.
+ - The sidebar structure that you see in our published docs is determined by the [`sidebars.js` file](https://github.com/OffchainLabs/arbitrum-docs/blob/master/website/sidebars.js) located at the root of the `website` directory.
+   - You may notice that our sidebars are currently organized using plain-old objects that specify individual files and folders.
+   - We're moving away from this "manual configuration" and towards convention-based [auto-generated sidebars](https://docusaurus.io/docs/sidebar/autogenerated).
+   - To ensure that this new approach works with your docs, **all net-new content should be organized within a folder structure that matches the desired sidebar structure**.
+
+
+<br/>
+
+## Frequently asked questions
+
+#### Can I point to my product from core docs? For example - if my product hosts a public RPC endpoint, can I add it to your [RPC endpoints and providers](/node-running/node-providers) page?
+
+These types of contributions are generally **not merged** unless they're submitted by employees of Offchain Labs.
+
+Instead of opening a PR for this type of contribution, click the `Request an update` button at the top of the published document to create an issue. Generally, third-party services are included in core docs only if we can confidently assert that the services are "**trustworthy, highly relevant to the core document at hand, and battle-tested by Arbitrum developers**" under a reasonable amount of scrutiny.
```

Comment:
> services are "**trustworthy, highly relevant to the core document at hand, and battle-tested by Arbitrum developers**"

Also, something like "highly relevant to the Arbitrum ecosystem" or something like that? In the sense that tools to include in the core docs must be tools that have proved to bring a lot of value to the ecosystem. Not 100% sure.

Reply:
IMO "core document at hand" is guaranteed to be "highly relevant to the Arbitrum ecosystem" - I'd say let's not block on this; we can clarify in response to demand for clarification from our readers.